ci-operator/step-registry/openshift-extended/test/openshift-extended-test-commands.sh

```diff
@@ -306,17 +306,21 @@ function run {
     fi
     # summarize test results
     echo "Summarizing test result..."
-    mapfile -t test_suite_failures < <(grep -r -E 'testsuite.*failures="[1-9][0-9]*"' "${ARTIFACT_DIR}" | grep -o -E 'failures="[0-9]+"' | sed -E 's/failures="([0-9]+)"/\1/')
-    failures=0
-    for (( i=0; i<${#test_suite_failures[@]}; ++i ))
-    do
-        let failures+=${test_suite_failures[$i]}
-    done
-    if [ $((failures)) == 0 ]; then
-        echo "All tests have passed"
-    else
-        echo "${failures} failures in openshift-extended-test" | tee -a "${SHARED_DIR}/openshift-e2e-test-qe-report-openshift-extended-test-failures"
+    failures=0 errors=0 skipped=0 tests=0
+    grep -r -E -h -o 'testsuite.*tests="[0-9]+"' "${ARTIFACT_DIR}" | tr -d '[A-Za-z=\"_]' > /tmp/zzz-tmp.log
+    while read -a row ; do
+        # if the last ARG of command `let` evaluates to 0, `let` returns 1
+        let errors+=${row[0]} failures+=${row[1]} skipped+=${row[2]} tests+=${row[3]} || true
+    done < /tmp/zzz-tmp.log
+
+    TEST_RESULT_FILE="${ARTIFACT_DIR}/test-results"
+    echo "failures: $failures, errors: $errors, skipped: $skipped, tests: $tests in openshift-extended-test" | tee -a "${TEST_RESULT_FILE}"
+    if [ $((failures)) != 0 ] ; then
+        echo "Failing Scenarios:" | tee -a "${TEST_RESULT_FILE}"
+        grep -h -r -E '^failed:' "${ARTIFACT_DIR}/.." | grep -v grep | cut -d'"' -f2 | sort -t':' -k2 | uniq | tee -a "${TEST_RESULT_FILE}"
```

Comment:
@liangxia better to add `|| true` because it is high possible to be failing for grep, cut, sort etc.
same comment for cucushift step.

Reply:
Not always true that it's high possible to fail for grep, cut, sort etc.
```
$ echo 1 2 3 | grep -h -E '^failed:' | grep -v grep | cut -d'"' -f2 | sort -t':' -k2 | uniq 
$ echo $?
0
```
Never mind, added `|| true` just in case.